docs/source/_static/js/custom.js

```diff
@@ -0,0 +1,259 @@
+// These three variables below need to be updated at each release for the selectors.
+
+const languages = [ "rust", "python", "node" ];
+// Last stable version for each language
+const stableVersion = {
+    "rust": "master",
+    "python": "v0.9.3",
+    "node": "master"
+}
+// Dictionary doc folder to Label for each language
+const versionMapping = {
+    "rust": {
+        "master": "master",
+    },
+    "python": {
+        "master": "master",
+        "v0.9.3": "v0.9.3"
```

Comment:
Sure, feel free to propose any idea/modification to this system. It could even serve for `transformers` and `datasets` that are using the same system!